pkg/istio-agent/sds-agent.go

```diff
@@ -448,7 +448,7 @@ func applyEnvVars() {
 	serverOptions.Pkcs8Keys = pkcs8KeysEnv
 	serverOptions.RecycleInterval = staledConnectionRecycleIntervalEnv
 	workloadSdsCacheOptions.SecretTTL = secretTTLEnv
-	workloadSdsCacheOptions.SecretRefreshGraceDuration = secretRefreshGraceDurationEnv
+	workloadSdsCacheOptions.SecretRefreshGracePeriodRatio = secretRefreshGracePeriodRatioEnv
```

Comment:
Where is SecretRefreshGracePeriodRatio used? I did a search in the PR and only found the places it is defined/declared, but not where it is used.

Reply:
It turned out to be a typo. Should be SecretRotationGracePeriodRatio.